src/info.cc

```diff
@@ -177,7 +177,7 @@ namespace
 ///////////////////////////////////////////////////////////////////
 namespace {
   // PoolQuery does no COW :( - build and return the basic query
-  inline PoolQuery printInfo_BasicQuery( Zypper & zypper, const PrintInfoOptions &options_r )
+  inline PoolQuery printInfo_BasicQuery( Zypper & zypper, const PrintInfoOptions &options_r)
```

Comment:
Space missing before close parenthesis according to coding style ;-)

Reply:
thanks for your comment, good catch!